coderd/database/queries/workspaceagentportshare.sql

```diff
@@ -1,13 +1,51 @@
 -- name: GetWorkspaceAgentPortShare :one
-SELECT * FROM workspace_agent_port_share WHERE workspace_id = $1 AND agent_name = $2 AND port = $3;
+SELECT
+	*
+FROM
+	workspace_agent_port_share
+WHERE
+	workspace_id = $1
+	AND agent_name = $2
+	AND port = $3;
```

Comment:
Is there a formatter I should be using but am somehow missing?

Reply:
Nope :( I just did it manually